pages/_app.js

```diff
@@ -125,8 +126,16 @@ class MyApp extends App {
 
   constructor(props) {
     super(props);
+    if (process.env.NODE_ENV === 'production') {
+      Fathom.load();
+      Fathom.setSiteId('IVMXEBVJ');
```

Comment:
Instead of hard-coded, can we add this to `next.config.js` (in the `env` section), then use that ENV variable here? And change the `if` to check for if this ENV variable is present? That makes it easier to test/debug, and easier to update in the future.